src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].name();
```

Comment:
you need to call `toLowerCase()` here as well

`name()` returns you a String representation of enum constant (`WHITE` instead of `white`)

```suggestion
        return Color.values()[index].name().toLowerCase();
```